standalone/content/js/standaloneRoomViews.jsx

```diff
@@ -788,6 +802,84 @@ loop.standaloneRoomViews = (function(mozL10n) {
     }
   });
 
+  var Slide = React.createClass({
+    propTypes: {
+      imageClass: React.PropTypes.string.isRequired,
+      indexClass: React.PropTypes.string.isRequired,
+      joinRoom: React.PropTypes.func.isRequired,
+      text: React.PropTypes.string.isRequired,
+      title: React.PropTypes.string.isRequired
+    },
+
+    handleGotItClick: function(event) {
+      event.stopPropagation();
+      event.preventDefault();
+      var overlayElem = document.getElementsByClassName("intro-overlay")[0];
+      overlayElem.classList.add("hide");
+      localStorage.setItem("introSeen", true);
+      this.props.joinRoom();
+    },
+
+    render: function() {
+      return (
+        <div className="slide">
+          <div className={this.props.indexClass}>
+            <div className="slide-layout">
+              <div className={this.props.imageClass} />
+              <h2>{this.props.title}</h2>
+              <div className="slide-text">{this.props.text}</div>
+              <button className="button-got-it" onClick={this.handleGotItClick}>
+                {mozL10n.get("intro_slide_button")}
+              </button>
+            </div>
+          </div>
+        </div>
+      );
+    }
+  });
+
+  var IntroOverlayView = React.createClass({
+    propTypes: {
+      joinRoom: React.PropTypes.func.isRequired
+    },
+
+
+    handleCloseClick: function(event) {
+      event.stopPropagation();
+      event.preventDefault();
+      // hide or remove the
+      var overlayElem = document.getElementsByClassName("intro-overlay")[0];
+      overlayElem.classList.add("hide");
+      this.props.joinRoom();
+    },
```

Comment:
I believe the semantics here should be identical to the "Got it" button (ie mark the slideshow as seen).  Feel free to check with Holly to be sure.  So this can probably share the handler with the Got It button.